test/unit/dialects/postgres/query-generator.test.js

```diff
@@ -1071,6 +1071,28 @@ if (dialect.startsWith('postgres')) {
         },
       ],
 
+      deleteQuery: [
+        {
+          arguments: ['myTable', { name: 'foo' }, { returning: true }, 'myTable'],
+          expectation: `DELETE FROM "myTable" WHERE "name" = 'foo' RETURNING *`,
+        }, {
+          arguments: ['myTable', { name: 'foo' }, { returning: false }, 'myTable'],
+          expectation: `DELETE FROM "myTable" WHERE "name" = 'foo'`,
+        }, {
+          arguments: ['myTable', { name: 'foo' }, { returning: ['*'] }, 'myTable'],
+          expectation: `DELETE FROM "myTable" WHERE "name" = 'foo' RETURNING *`,
```

Comment:
`*` should not have special meaning in the `returning` array. It's valid in PG to name a column "*" (unlikely, yes).

This query should produce this:

```suggestion
          expectation: `DELETE FROM "myTable" WHERE "name" = 'foo' RETURNING "*"`,
```

`returning: true` will be used for `RETURNING *`